### PR TITLE
[#113159997] Add tests for concourse manifest generation

### PR DESCRIPTION
**Pull request by saliceti**
## What
Story: [Generate mbus passwords for Deployer Concourse and BOSH](https://www.pivotaltracker.com/story/show/113159997)

The concourse manifest generation was refactored in https://github.com/alphagov/paas-cf/pull/142. This adds tests to finalise it.

## How to review
Load the ruby environment related to the paas-cf Gemfile and run the tests via `make test`

## Who can review
Anyone but me

### CODE REVIEW

File: manifests/concourse-manifest/spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+# Requires supporting ruby files with custom matchers and macros, etc, in
+# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
+# run as spec files by default. This means that files in spec/support that end
+# in _spec.rb will both be required and run as specs, causing the specs to be
+# run twice. It is recommended that you do not name files matching this glob to
+# end with _spec.rb. You can configure this pattern with the --pattern
+# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
+#
+puts File.expand_path("../support/**/*.rb", __FILE__)
```
Comment (alext) [inline manifests/concourse-manifest/spec/spec_helper.rb]: Did you mean to leave this `puts` here?

**Comment by saliceti** [comment]
@alext fixed, thanks.

**Comment by combor** [comment]
testing. please ignore

**Comment by keymon** [comment]
LGTM, it does not test every case, but it sets a good base to add more tests and makes the directory structure more homogenous.

**Pull request merged**